docs/DeveloperGuide.md

```diff
@@ -18,12 +44,162 @@
+    <img src="images/RemoveStatCommand.png">
+    <br />
+    <i>Figure 7: Sequence Diagram for Remove Pet Stat Command</i>
+</p>
+
 ## User Stories
 
-|Version| As a ... | I want to ... | So that I can ...|
-|--------|----------|---------------|------------------|
-|v1.0|new user|see usage instructions|refer to them when I forget how to use the application|
-|v2.0|user|find a to-do item by name|locate a to-do without having to go through the entire list|
+| Version | As a ...         | I want to ...                                           | So that I can ...                                        |
+|---------|------------------|---------------------------------------------------------|----------------------------------------------------------|
+| v1.0    | basic user       | add new pets                                            | track more pets                                          |
+| v1.0    | basic user       | record my pet's weight                                  | track if the weight is healthy                           |
+| v1.0    | basic user       | record my pet's age                                     | track how old my pet is                                  |
+| v1.0    | basic user       | remove a pet from my list                               | declutter the petlist from pets I no longer take care of |
+| v1.0    | large-scale user | see basic info on all my pets in one place              | get an overview of all my pets                           |
+| v2.0    | basic user       | save my pet list                                        | save my pet list data when i close the program           |
+| v2.0    | basic user       | load my saved pet list                                  | resume where i left off previously                       |
+| v2.0    | basic user       | add task                                                | remind myself to carry out a task                        |
+| v2.0    | basic user       | mark uncompleted tasks as done                          | check off tasks that i complete                          |
+| v2.0    | basic user       | unmark completed tasks                                  | uncheck off tasks that i did not complete                |
+| v2.0    | basic user       | see all my tasks in one place                           | get an overview of all my tasks                          |
+| v2.0    | basic user       | edit my pet information                                 | modify my pet information when i need to update it       |
+| v2.0    | basic user       | edit my task information                                | modify my task information when i need to update it      |
+| v2.0    | basic user       | get a schedule of what time tasks must be done          | better meet my pets need by carrying out the task        |
+| v2.0    | forgetful user   | receive reminders about what task should have been done | complete due tasks that i forgot to do                   |
 
```

Comment:
Kindly update the non-functional requirements, glossary and instruction on manual testing with a small draft for a clearer idea
![image](https://user-images.githubusercontent.com/80743735/228765242-ba3d6053-4e63-48c5-bb56-f0598b736dc7.png)